coursedata/adventures/de.yaml

```diff
@@ -603,164 +609,195 @@ adventures:
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
 
                     ```
-                    print Drawing figures
-                    angle is 90
-                    turn angle
+                    print Zeichne Formen
+                    Winkel is 90
+                    turn Winkel
                     forward 25
-                    turn angle
+                    turn Winkel
                     forward 25
                     ```
                 start_code: |
-                    print Turtle race!
-                    hoek is 90
-                    turn hoek
+                    print Schildkrötenrennen!
+                    Winkel is 90
+                    turn Winkel
                     forward 25
             3:
                 story_text: |
-                    ## Searching turtle
-                    In this level you can use use `random` with the drawing turtle. A random choice makes the turtle walk a different path each time.
-                    Use `at random` to choose a value from a list. You can copy and paste lines 2 and 3 to create a longer random path.
+                    ## Suchende Schildkröte
+                    In diesem Level kannst Du den Befehl `random` nutzen, während die Schildkröte etwas zeichnet.
+                    Durch eine zufällige Wahl sieht die Zeichnung jedes Mal anders aus.
+                    Nutze `at random` um einen zufälligen Wert einer Liste zu wählen.
+                    Wenn Du die Zeilen 2 und 3 kopierst und wieder einfügst, kannst Du einen längeren Pfad erzeugen.
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
 
                     ```
-                    angles is 10, 50, 90, 150, 250
-                    turn angles at random
+                    Winkel is 10, 50, 90, 150, 250
+                    turn Winkel at random
                     forward 25
                     ```
 
                 start_code: |
-                    angles is 10, 50, 90, 150, 250
-                    turn angles at random
+                    Winkel is 10, 50, 90, 150, 250
+                    turn Winkel at random
                     forward 25
             4:
                 story_text: |
-                    ## Let's draw
-                    In level 4 you have to use quotation marks with `print` and `ask`. Also when drawing!
+                    ## Lass uns zeichnen
+                    In Level 4 musst Du nun auch die Anführungszeichen bei `print` und `ask` nutzen.
+                    Das gilt auch für's Zeichnen!
+
+                example_code: |
+                    ## Hedy Beispiel-Code
 
-                    ## Example Hedy code
                     ```
-                    print 'Drawing figures'
-                    angle is 90
-                    turn angle
+                    print 'Zeichne Figuren:'
```

Comment:
I think "Formen" captures the meaning better than "Figuren". Also, a colon probably doesn't make much sense when no further textual terminal output follows.